src/ParallelAlgorithms/SurfaceFinder/Python/FindRadialSurface.py

```diff
@@ -169,6 +169,7 @@ def find_radial_surface(
                         functions_of_time,
                     )
                 )[:, 0]
+                - initial_guess.expansion_center
```

Comment:
Update the test to be off-center to catch this?

Reply:
Didn't do it because the `Sphere` domain creator can't do off-center coords at the moment